SimEngine.py

```diff
@@ -96,16 +99,28 @@ def runToCompletion(self,startFunc):
         
         return self._currentTime
     
-    def schedule(self,ts,cb,selfDestruct=False):
+    def schedule(self,ts,cb,tag='None'):
```

Comment:
here `'None'` is the string "None", what you want is to null value in Python, `None`. So `tag=None`.